tech/languages/csharp/csharp.md

```diff
@@ -0,0 +1,51 @@
+---
+title: C#
```

Comment:
Titles should be unique in one subsection. It would not make sense otherwise(they form a link/tab/row in left menu on every subsection page).